File: Beginner/3342 - Keanu/3342.cs
using System;

class URI
{
    static void Main(string[] args)
    {
        int N, cBranca, cPreta;
        N = int.Parse(Console.ReadLine());

        int totalCells = N * N;
        cPreta = totalCells / 2;
        cBranca = totalCells - cPreta;

        Console.WriteLine($"{cBranca} casas brancas e {cPreta} casas pretas");
    }
}

File: Beginner/3342 - Keanu/3342.c
#include <stdio.h>

int main() {
    int N, cBranca, cPreta;
    scanf("%i", &N);

    int totalCells = N * N;
    cBranca = totalCells / 2;
    cPreta = totalCells - cBranca;

    printf("%i casas brancas e %i casas pretas", cBranca, cPreta);

    return 0;
}